release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -177,6 +177,16 @@ For more information, see link:https://kubernetes.io/docs/concepts/services-netw
 
 [id="ocp-4-11-osdk"]
 === Operator development
+==== Performance Addon Operator functions moved to the Node Tuning Operator
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. The Node Tuning Operator is part of the standard installation for {product-title} 4.11. If you upgrade to {product-title} 4.11, the Node Tuning Operator removes the Performance Addon Operator, and all related artifacts, on startup. 
+
+[NOTE]
+====
+The Performance Profile Creator still uses the Performance Addon Operator `must gather` image.
```

Comment:
Maybe this is a little bit confusing.
PPC do not use the same image as `must-gather`.

PPC uses PAO image (`performance-addon-operator`) with a different entry point while must-gather uses its own image (`performance-addon-operator-must-gather`)

Reply:
Thanks @jlojosnegros - I updated the note. Less confusion now hopefully?